### ISSUE TITLE
fortran only flags solution for clover mpi vs fortran arg mismatch?

### ISSUE THREAD

**Issue by cyrush** [Alpine-DAV/ascent#969]
```
    # these flags are needeed for gfortran 10 to avoid
    # compile errors with the mpi interface
    if(CMAKE_Fortran_COMPILER_VERSION VERSION_GREATER_EQUAL "10" AND Fortran_COMPILER_FAMILY_IS_GNU)
        add_compile_options($<$<COMPILE_LANGUAGE:Fortran>:-fallow-invalid-boz>
                            $<$<COMPILE_LANGUAGE:Fortran>:-fallow-argument-mismatch>
                           )
    endif()
```
?

**Comment by cyrush** [comment]
Note: also add FORTRAN flag filtering for blt, like:

https://github.com/LLNL/conduit/pull/1063

**Comment by cyrush** [comment]
resolved in #1057 and #1076
